src/test/java/com/sap/sgs/phosphor/fosstars/model/score/oss/VulnerabilityDiscoveryAndSecurityTestingScoreTest.java

```diff
@@ -0,0 +1,37 @@
+package com.sap.sgs.phosphor.fosstars.model.score.oss;
+
+import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.VULNERABILITIES;
+import static com.sap.sgs.phosphor.fosstars.model.value.Vulnerability.Builder.newVulnerability;
+
+import com.sap.sgs.phosphor.fosstars.model.Value;
+import com.sap.sgs.phosphor.fosstars.model.other.Utils;
+import com.sap.sgs.phosphor.fosstars.model.value.ScoreValue;
+import com.sap.sgs.phosphor.fosstars.model.value.Vulnerabilities;
+import java.util.Date;
+import java.util.Set;
+import org.junit.Test;
+
+public class VulnerabilityDiscoveryAndSecurityTestingScoreTest {
+
+  private static ProjectSecurityTestingScore SECURITY_TESTING_SCORE
```

Comment:
Is it ok to use the same object while testing in different methods?

Reply:
As long as the object is immutable, or at least has an expected state before before a test case runs, I think yes, it's okay.